ui/package.json

```diff
@@ -124,6 +124,7 @@
     "ember-modifier": "^3.1.0",
     "ember-page-title": "^6.2.2",
     "ember-power-select": "^5.0.3",
+    "ember-qrcode-shim": "^0.4.0",
```

Comment:
Jordan I'll leave some additional comments in slack about this, but for now, this solves the qrCode issue. It's a wrapped that goes around the qrCode.js library, which ALSO hasn't been updated in a long time, but is still recommended as the way to handle qr codes.

Reply:
This looks good to me. Let's roll with it if it suits our needs and doesn't introduce deprecation warnings or those type of things in the build.